main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -1747,6 +1748,8 @@ object BackendObjType {
 
     def ApplyMethod: InterfaceMethod = InterfaceMethod(this.jvmName, "applyFrame", mkDescriptor(Value.toTpe)(Result.toTpe))
 
+    def DirectApply(defn: Def)(implicit root: ReducedAst.Root): StaticMethod = StaticMethod(this.jvmName, JvmName.DirectApply, MethodDescriptor(defn.fparams.map(fp => BackendType.toBackendType(fp.tpe)), BackendObjType.Result.toTpe))
```

Comment:
This method insn't in the Frame class so it shouldn't be under the Frame object. You can move it to GenFunAnd...